paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEvent.kt

```diff
@@ -491,6 +503,11 @@ internal sealed class PaymentSheetEvent : AnalyticsEvent {
     }
 }
 
+internal enum class LinkMode {
```

Comment:
Maybe this should be a top level type, and we should put the functionality here for `analyticsValue`, and getting it from elements sessions, so we can unit test it.